crates/lox_core/src/bodies.rs

```diff
@@ -4,15 +4,15 @@ pub mod planets;
 pub mod satellites;
 pub mod sun;
 
-pub trait NaifId {
+pub trait NaifId: Copy {
```

Comment:
I've not seen this syntax before. Are you just looking for a derive Copy?

Reply:
This is supertrait syntax. I am telling the compiler that implementers of `NaifId` also need to implement `Copy`. Alternatively, I could (probably) change this function signature to:
```rust
pub fn naif_id<T: NaifId + Copy>(_: T) -> i32 {
    <T as NaifId>::id()
}
```